### DISCUSSION TITLE
EvaluateScriptAsync  calling a function already loaded in a vue html page

### DISCUSSION THREAD

**Discussion by dlugolinski13** [cefsharp/CefSharp#4384] [General]
I am trying to call a JavaScript function that is already part of the loaded scripts (vuejs pages bundle.js) from the C# browser.EvaluateScriptAsync()

browser.EvaluateScriptAsync("functionInLoadedScripts()");

Is this possible?
I can execute a script that is simple but it shows executing in blank...

**Comment by dlugolinski13** [comment]
Solved on my own...

**Reply by Blubberblub** [reply]
@dlugolinski13 Can you elaborate? Having a similar issue.
